x-pack/metricbeat/module/cockroachdb/status/manifest.yml

```diff
@@ -0,0 +1,6 @@
+default: true
+input:
+  module: prometheus
```

Comment:
LGTM for now but I would really like to see this changing to `type: prometheus` one day, meaning having real inputs.

Reply:
Yes, a possible future step after all this is to add some kind of inputs registry for implementations that don't have enough entity to be metricsets, but I didn't want to open this door yet :slightly_smiling_face:

Reply:
++